src/constants.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+exports.PROTOCOL_MULTICODEC = '/rendezvous/1.0.0'
```

Comment:
This spec doesn't mention this protocol at all, we should clarify this there and that this protocol runs over libp2p streams.